slack_sdk/oauth/installation_store/file/__init__.py

```diff
@@ -129,8 +129,8 @@ def find_bot(
                 data = json.loads(f.read())
                 return Bot(**data)
         except FileNotFoundError as e:
-            message = f"Failed to find bot installation data for enterprise: {e_id}, team: {t_id}: {e}"
-            self.logger.warning(message)
+            message = f"Any bot installation data was found for enterprise: {e_id}, team: {t_id}: {e}"
```

Comment:
Did you mean “No” instead of “Any” here (and in line 175)?

I’m thinking something like:
```
f”No bot installation data found for enterprise {e_id}, team {t_id}: {e}"
```

Reply:
Oops, yes, I meant "No" but I like your suggestion. Will update. Thanks!